_docs/_latest/setup/install.md

```diff
@@ -24,11 +24,196 @@ organization Cloud IAM policy.
 
 ## Setting up Forseti Security
 
-Users can install Forseti using Terraform by clicking the button below.
+The Forseti Terraform module is the only supported method of installing Forseti Security. The default infrastructure for 
+Forseti is Google Compute Engine. This module also supports installing Forseti on Google Kubernetes Engine (GKE), 
+and at some point in the future will become the default. For more information on installing Forseti on-GKE, please see 
+the [detailed guide on setting up Forseti on-GKE]({% link _docs/latest/setup/forseti-on-gke.md %}).
 
-[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fforseti-security%2Fterraform-google-forseti.git&cloudshell_git_branch=master&cloudshell_working_dir=examples/install_simple&cloudshell_image=gcr.io%2Fgraphite-cloud-shell-images%2Fterraform%3Alatest&cloudshell_tutorial=.%2Ftutorial.md)
+## Google Cloud Shell Walkthrough
+A Google Cloud Shell walkthrough has been setup to make it easy for users who are new to Forseti and Terraform. 
+This walkthrough provides a set of instructions to get a default installation of Forseti setup that can be used in a 
+production environment.
 
-Documentation on installing can be found [here](https://registry.terraform.io/modules/terraform-google-modules/forseti/google).
+If you are familiar with Terraform and would like to run Terraform from a different machine, you can skip this 
+walkthrough and move onto the How to Deploy section below.
+
+[![Open in Google Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fforseti-security%2Fterraform-google-forseti.git&cloudshell_git_branch=module-release-5.0.0&cloudshell_working_dir=examples/install_simple&cloudshell_image=gcr.io%2Fgraphite-cloud-shell-images%2Fterraform%3Alatest&cloudshell_tutorial=.%2Ftutorial.md)
+
+## How to Deploy
+In order to run this module you will need to be authenticated as a user that has access to the project and can 
+create/authorize service accounts at both the organization and project levels. To login to GCP from a shell:
+
+```bash
+gcloud auth login
+```
+
+### Install Terraform
+Terraform version 0.12 is required for this module, which can be downloaded from the 
+[Terraform website](https://www.terraform.io/downloads.html).
+
+### Create the Service Account and enable required APIs
+
+#### Service Account
```

Comment:
We have mentioned elsewhere that if someone is org admin, they don't need to create a separate service account.  Can we do mention that here too?

Reply:
@blueandgold added in- please take a look!